alacritty_terminal/src/config/mod.rs

```diff
@@ -17,132 +16,103 @@ pub use crate::config::bell::{BellAnimation, BellConfig};
 pub use crate::config::colors::Colors;
 pub use crate::config::scrolling::Scrolling;
 
-pub const LOG_TARGET_CONFIG: &str = "alacritty_config";
-const DEFAULT_CURSOR_THICKNESS: f32 = 0.15;
-const MAX_SCROLLBACK_LINES: u32 = 100_000;
+pub const LOG_TARGET_CONFIG: &str = "alacritty_config_derive";
```

Comment:
Currently this is hardcoded, I'm not sure there's any good way to share this constant with `alacritty_config_derive`. The only option would be to import it from `alacritty_config_derive`, but that would create a cyclic dependency.

Unfortunately it's not possible to export constants from proc macros.